### ISSUE TITLE
Permission denied to bind lan-mouse-socket

### ISSUE THREAD

**Issue by xf-** [feschber/lan-mouse#66]
Can't find anything about it in issue tracker or README.

Tired to connect via `cargo run --release -- --frontend cli`
```
INFO  lan_mouse::config] using config: "config/lan-mouse/config.toml"
INFO  lan_mouse] Press Ctrl+Alt+Shift+Super to release the mouse
ERROR lan_mouse] failed to bind lan-mouse-socket: Permission denied (os error 13)
```

- `config.toml` only sets a port and first tries I had no config and same error.
- Used barrier so far and stop it.
- Can't find an open port, and it is not used (also tried different ports)

Running Wayland with latest commit on remote side. X11 on client side

**Comment by feschber** [comment]
The error is about the internal Unix Domain Socket that is communicating between the GUI / CLI and the service.
So port should not matter here.

Can you tell me what `$XDG_RUNTIME_DIR` points to on your system? (Thats where the socket is created)
And can you manually create files in that directory?

**Comment by xf-** [comment]
It was on 0 for what ever reason. Now it is correct.

Now it complains about libei not being available (installed 1.2.0) and UnknownMethod ConnectToEIS (xdg desktop portal is installed).
Try to find the issue and get it to work :)
